core/unix/os.c

```diff
@@ -762,7 +762,7 @@ our_init(int argc, char **argv, char **envp)
     return 0;
 }
 
-#if defined(STATIC_LIBRARY) || defined(STANDALONE_UNIT_TEST)
+#if defined(STATIC_LIBRARY) || defined(STANDALONE_UNIT_TEST) || defined(RISCV64)
```

Comment:
Please update the comment because as is it doesn't make sense why you'd add RISCV64 to the defines above.

Reply:
Added comments for this change in https://github.com/DynamoRIO/dynamorio/pull/6095/commits/b127d66452df89e4ec84e81d7c04bbd09cf5937f.